vmdb/app/models/ems_kubernetes.rb

```diff
@@ -3,6 +3,7 @@ class EmsKubernetes < EmsContainer
   has_many :container_groups,                     :foreign_key => :ems_id, :dependent => :destroy
   has_many :container_services,                   :foreign_key => :ems_id, :dependent => :destroy
   has_many :container_replication_controllers,    :foreign_key => :ems_id, :dependent => :destroy
+  has_many :container_namespaces,                 :foreign_key => :ems_id, :dependent => :destroy
```

Comment:
> We'll have to sync with the openshift patch on this one. It's not worth having 2 different tables (container_projects and contaner_namespaces) that are essentially the same and hold almost entirely the same data.

I'd like to understand your proposal better. Please share your thoughts on the following points:

1. for openshift provider inventory do you propose to :
option A - persist a new entity that is a merge of namespace and project? if yes - which entity type, and what if some properties of a namespace and a project contradict each other?

option B - persist both namespace and project as 2 models/2 records in the same table? (STI)
What about the extra attributes that project has that namespace doesn't? Should those just be empty for all namespaces?

option C - persist only the project entity?

2. What about namespaces that aren't backed with a project entity? How does that work with the above options?

3. How do you propose to present this in UI given that it is the same UI for both k8s and opeshift entities? How its menu supposed to be called in the UI? Please keep in mind, It can't be called both namespace and project, and it can't change dynamically based on provider time because it is always visible.
(also worth to note, some manageiq environments might have only k8s providers without openshift)

4. How do you envision the openshift provider relatioships? should it even have the relationship to namespace  listed on Relationships, or just to projects?

5. How do you propose to keep the relationship of project to entities?
Most of k8s entities 'belong' to a namespace (still not implemented on our side). Should those in your opinion be changed to belong to project in openshift provider? or k8s stays "with" namespace, and ose entities belong to a project? (the latter means there must still be both types of entity - namespace and project)